.circleci/config.yml

```diff
@@ -133,6 +156,97 @@ jobs:
       - run:
           name: Run pylint
           command: ./tools/pylint.sh
+  webpack:
+    docker:
+      - image: "cimg/node:lts"
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Compile and bundle CSS and JS
+          command: npm run prod
+      - persist_to_workspace:
+          root: .
+          paths:
+            - integreat_compass/static/dist
+            - integreat_compass/webpack-stats.json
+  build-package:
+    docker:
+      - image: cimg/python:3.9.16
+    resource_class: small
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Enable virtual environment
+          command: echo "source .venv/bin/activate" >> $BASH_ENV
+      - run:
+          name: Use alternative README.md file
+          command: mv integreat-compass/README.md . 
+      - run:
+          name: Build integreat-compass package
+          command: python3 -m build
+      - persist_to_workspace:
+          root: .
+          paths:
+            - dist
+  bump-dev-version:
+    docker:
+      - image: cimg/python:3.9.16
+    resource_class: small
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Enable virtual environment
+          command: echo "source .venv/bin/activate" >> $BASH_ENV
+      - run:
+          name: Bump version
+          command: |
+            # Install recent version of pip
+            echo "Upgrade pip to make sure 'pip index' is available"
+            pip install --upgrade pip
+            # Check which versions of integreat-compass are available on the TestPyPI repository
+            AVAILABLE_VERSIONS=$(pip index versions integreat-compass --pre -i https://test.pypi.org/simple/)
+            echo "Current available versions on TestPyPI: ${AVAILABLE_VERSIONS}"
+            CURRENT_ALPHA_VERSION=$(echo "${AVAILABLE_VERSIONS}" | head -n 1)
+            echo "Most recent version on TestPyPI: ${CURRENT_ALPHA_VERSION}"
+            CURRENT_ALPHA_VERSION=$(echo "${CURRENT_ALPHA_VERSION}" | sed "s/integreat-compass (\([^()]*\)a0)/\1-alpha/")
+            echo "Version converted to alternative format: ${CURRENT_ALPHA_VERSION}"
+            # Get current prod version
+            CURRENT_VERSION=$(python -c "import integreat_compass; print(integreat_compass.__version__)")
+            echo "Current production version: ${CURRENT_VERSION}"
+            # Bump version to current alpha version if it is newer
+            # Attention: exit(True) in Python means exit(1) which is False in Bash :)
+            if python -c "from looseversion import LooseVersion; exit(LooseVersion('${CURRENT_VERSION}') > LooseVersion('${CURRENT_ALPHA_VERSION}'))"; then
```

Comment:
You need to add `looseversion` to the dev dependencies in order for this to work.